CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 **Fixed**:
 
+- **decidim-core**: Fix editing a process after an error.[\#2653](https://github.com/decidim/decidim/pull/2653)
```

Comment:
Changelog should be in `decidim-assemblies` and `decidim-participatory_processes`, since that's what you're modifying (note core)